### PR TITLE
Add example to API client usage doc

### PR DESCRIPTION
**Pull request by jaegeral**
Introduce a small doc how to use return_fields in the api client.

### CODE REVIEW

File: docs/APIClient.md [hunk 1]
```diff
@@ -150,6 +150,13 @@ data = sketch.explore('google.com', as_pandas=True)
 
 This will return back a pandas DataFrame with the search results.
 
+Another example search query:
+```
+data = sketch.explore('192.168.0.1 AND NOT timestamp_desc:foobar', as_pandas=True,return_fields='datetime,message,timestamp_desc')
```
Comment (kiddinn) [inline docs/APIClient.md:155]: can you make this

```
data = sketch.explore(
    '192.168.......',
    as_pandas=True,
    return_fields='.....'
)
```

just to make it easier to read

**Review by kiddinn — COMMENTED** [review]
one minor comment, plus the name of the PR is not very descriptive.... can you change that?

File: docs/APIClient.md (hunk 1 above)
Comment (jaegeral) [reply docs/APIClient.md]: sure thing - done

and renamed the PR

**Review by kiddinn — APPROVED** [review]
LGTM

**Pull request merged**